ansible/roles/nethserver/files/nethserver-networks-init

```diff
@@ -1,8 +1,38 @@
-#! /bin/bash
+#!/bin/bash
 
+#Configure link-local addrress
+ifconfig eth0 169.254.$(($RANDOM % 255)).$((1 + $RANDOM % 253)) netmask 255.255.0.0 up
```

Comment:
This is quite scary. I guess it's needed to use curl. Am I right?

Reply:
Yes, at that moment the instance doesn't have any valid network configuration, and this is the standard way to select a IPv4 Link-Local address:
https://tools.ietf.org/html/rfc3927#section-2.1
>   When a host wishes to configure an IPv4 Link-Local address, it
   selects an address using a pseudo-random number generator with a
   uniform distribution in the range from 169.254.1.0 to 169.254.254.255
   inclusive.

But I just notice that I have selected a too large interval, I will fix that.